pageserver/src/http/routes.rs

```diff
@@ -3047,6 +3140,10 @@ pub fn make_router(
             "/v1/tenant/:tenant_shard_id/timeline/:timeline_id/layer/:layer_file_name",
             |r| api_handler(r, evict_timeline_layer_handler),
         )
+        .post(
+            "/v1/tenant/:tenant_shard_id/timeline/:timeline_id/layer/:layer_name/scan_disposable_keys",
```

Comment:
I presume this is not a testing-only API because some piece of automation in the field is going to use it?  Can you add a comment to the handler definition that mentions the use case?

Reply:
no, just copy-pasta, will make it a testing api